ibm/mas_devops/playbooks/cp4d/create-db2-instance.yml

```diff
@@ -25,6 +28,16 @@
     db2wh_backup_storage_class: "{{ lookup('env', 'DB2WH_BACKUP_STORAGE_CLASS') }}"
     db2wh_backup_storage_size_gb: "{{ lookup('env', 'DB2WH_BACKUP_STORAGE_SIZE_GB') | default(100, true) }}"
 
+    # Configure CP4D transaction logs storage .. despite the name, this ONLY applies to Db2 in CP4D 4.0
```

Comment:
This TODO comment can be removed, it's already been done

Reply:
Fixed.

Reply:
reviewed